boards/itsybitsy_m4/examples/blinky_basic.rs

```diff
@@ -9,7 +9,7 @@ use panic_halt as _;
 #[cfg(feature = "use_semihosting")]
 use panic_semihosting as _;
 
-use bsp::entry;
+use bsp::*;
```

Comment:
I am personally not a fan of glob imports and it doesn't match most of the other examples. Consider aligning the import style